### PR TITLE
Fix registering host xref

### PR DESCRIPTION
**Pull request by ryandeussing**
Satellite validation is failing b/c this xref is to a file that the title does not include

Cherry-pick into:

* [ ] Foreman 3.0
* [x] Foreman 2.5 (Satellite 6.10)
* [ ] Foreman 2.4
* [ ] Foreman 2.3 (Satellite 6.9)
* [ ] Foreman 2.1 (Satellite 6.8)

### CODE REVIEW

**Comment by ryandeussing** [comment]
@adahms thanks for this tip as well!

File: guides/common/modules/proc_installing-the-satellite-server-packages.adoc [hunk 1]
```diff
@@ -32,7 +32,7 @@ include::proc_installing-server-packages-el.adoc[]
 
 endif::[]
 
-ifdef::foreman-el,katello[]
+ifdef::foreman-el,katello,satellite[]
 == [[rhel-7]]Red Hat Enterprise Linux 7
```
Comment (lzap) [inline guides/common/modules/proc_installing-the-satellite-server-packages.adoc:36]: This is a valid hunk.

File: guides/doc-Managing_Hosts/topics/Registering_Hosts.adoc [hunk 2]
```diff
@@ -22,7 +22,6 @@ You can also register Atomic Hosts to {ProjectServer} or {SmartProxyServer}.
 
 Use one of the following procedures to register a host:
 
-* xref:registering-a-host-to-project-using-the-global-registration-template_managing-hosts[]
```
Comment (lzap) [inline guides/doc-Managing_Hosts/topics/Registering_Hosts.adoc:25]: However this is not valid, your PR is for the `master` branch and in there, this is a valid link.

If you need this to be fixed for Satellite 6.10, then file a PR against 2.5 branch, not master. We cannot merge this just like that.

Comment (ryandeussing) [reply guides/doc-Managing_Hosts/topics/Registering_Hosts.adoc:25]: Thanks, understood.

**Comment by lzap** [comment]
I suggest you split this into two PRs, the missing "satellite" if statement looks good. You need to rebase it on top of master tho.

The other file as another PR for 2.5 only branch. But explain in more detail what is wrong so I can check it first.

File: guides/common/modules/proc_installing-the-satellite-server-packages.adoc (hunk 1 above)
Comment (lzap) [reply guides/common/modules/proc_installing-the-satellite-server-packages.adoc:36]: Oh was this merged already: https://github.com/theforeman/foreman-documentation/pull/722

**Comment by ryandeussing** [comment]
Changing the base blew this PR up. I will close, apply the same changes to a branch off of 2.5, and reopen a PR against 2.5.

**Pull request closed without merging**